spec/fleiss/worker_spec.rb

```diff
@@ -16,6 +16,13 @@
     runner.kill
   end
 
+  around do |example|
+    callback = ->(*args) { notifications.push ActiveSupport::Notifications::Event.new(*args) }
```

Comment:
Same here - can this take just one arg or it's not applicable to lambdas?

Reply:
It's not works for `subscribed` method and callback, please lets keep this in tests